### DISCUSSION TITLE
Nodes not Synchronising Time

### DISCUSSION THREAD

**Discussion by okdlukas** [okd-project/okd#1945] [Q&A]
On a new deployed Cluster i have the problem of Nodes that are not synchronizing Time. I looked into that problem and it seems all of my Master and worker nodes are infected. 

I looked into the solution and found something in the documentation like : Optional: Configuring NTP for disconnected clusters. 
The Examples there with butane do not work as butane throws some `Error translating config: error parsing version; must be a valid semver` messages. 

I am not sure if that is really the problem and don't know what i could check to find out.

**Comment by rafa13o** [comment]
Hello!

I don't know if you've found the solution yet, but it's listed here:
[Butane config in example cannot be translated: error parsing version; must be a valid semver](https://github.com/openshift/openshift-docs/issues/74876)

The explanation is that according to the butane documentation (https://coreos.github.io/butane/upgrading-openshift/#from-version-4170-to-4180), as there have been no configuration changes since version 4.14.0, they are ‘stuck’ on that version (or so it seems), so when you create your butane file, the version you should put is 4.14.0.
